### PR TITLE
Add code of conduct link.

### PR DESCRIPTION
**Pull request by afshin**
Fixes https://github.com/jupyterlab/extension-examples/issues/83

### CODE REVIEW

**Comment by afshin** [comment]
Whoa. A lot of CI failures.

**Comment by jtpio** [comment]
It was mentioned in another issue that updating to `actions/checkout@v2` should help.

**Comment by jtpio** [comment]
See https://github.com/jupyterlab/extension-examples/pull/85.

If it passes, we can merge #85 and rebase #84.

File: README.md [hunk 1]
```diff
@@ -263,3 +263,9 @@ Complementary to these examples, you can rely on the official JupyterLab documen
 ## Credits
 
 We would like to thank [MMesch](https://github.com/MMesch) for [initiating this work](https://github.com/MMesch/labextension_tutorial), as well as everyone else who contributed!
+
+
+## Community Guidelines and Code of Conduct
+
+This examples repository is Jupyter project and follows the Jupyter
```
Comment (jtpio) [inline README.md]: Missing "a"?

```suggestion
This examples repository is a Jupyter project and follows the Jupyter
```

**Comment by jtpio** [comment]
Looks like the root README file needs to be formatted with prettier: https://github.com/jupyterlab/extension-examples/pull/84/checks?check_run_id=522713860

**Pull request merged**